File: APP/src/app/model/pessoa.ts
enum Sexo {
    MASCULINO, FEMININO
}

export class Pessoa {
    idPessoa: number;
    nome: string;
    nascimento: string;
    telefone: string;
    cpf: string;
    idade: number;
    sexo: Sexo | null;

    constructor() {
        this.idPessoa = 0;
        this.nome = "";
        this.nascimento = "";
        this.telefone = "";
        this.cpf = "";
        this.idade = 0;
        this.sexo = null;
    }
}

File: JAVA/src/main/java/com/inf_112/trabalho_final/model/Pessoa.java
package com.inf_112.trabalho_final.model;

import java.sql.Date;
import lombok.Data;

@Data
public class Pessoa {
    public enum Sexo {
        MASCULINO, FEMININO
    }

    private int idPessoa;
    private String nome;
    private Date nascimento;
    private String telefone;
    private String cpf;
    private int idade;
    private Sexo sexo;
}
